### PR TITLE
A/B Test: Privacy protection auto added

### PR DESCRIPTION
**Pull request by aidvu**
- Auto add privacy protection
- Change checkbox to a radio button
- Remove the privacy popup

Before:
privacy old
After:
image

Test:
- Domains -> Add
- Select a domain
- Make sure it's auto-added
- Make sure the privacy form on details form works

Test Signup:
- http://calypso.localhost:3000/start/
- Get to the domains screen
- Pick a domain with plan
- Make sure privacy is added

Test /domains:
- calypso.localhost:3000/start/domain-first?new=testing-privacy-auto-add-hehe-666.blog
- Just buy a domain
- make sure privacy is auto added

### CODE REVIEW

File: client/my-sites/checkout/checkout/privacy-protection.jsx [hunk 1]
```diff
@@ -99,10 +103,132 @@ class PrivacyProtection extends Component {
...
+														strong: <strong />,
+													},
+												}
+											)
+										}
+									</span>
+									<span className={ 'checkout__privacy-protection-radio-price-text' }>
+										{
+											translate(
+												'%(cost)s/year',
```
Comment (a8ci18n) [inline client/my-sites/checkout/checkout/privacy-protection.jsx]: Hi! I've found a possible matching string that has already been translated **24** times:
`translate( '%(cost)s per year', '%(cost)s per domain per year' )` *ES Score: 17*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by aidvu** [comment]
Resolves: https://github.com/Automattic/wp-calypso/issues/12622

**Comment by mikeshelton1503** [comment]
Also fixes #12632

**Comment by lucasartoni** [comment]
It looks great to me.

**Comment by mikeshelton1503** [comment]
LGTM 👍 

I added a section header to the Privacy Protection card to give it a little more prominence and consistency. If that looks good I think this is ready to go!

**Comment by alisterscott** [comment]
This PR was reverted as it broke all paid sign up flows during the processing step:

```step-actions.js:194 Uncaught TypeError: Cannot read property 'meta' of undefined
    at addToCartAndProceed (step-actions.js:194)
    at eval (step-actions.js:219)
    at eval (send-request.js:104)
    at XMLHttpRequest.xhrOnLoad (index.js:177)
    at resolve (index.js:547)
    at onmessage (index.js:510)```

**Pull request merged**